gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,7 @@
-#Sun May 28 17:31:34 MSK 2017
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.3-all.zip
-distributionSha256Sum=71a787faed83c4ef21e8464cc8452b941b5fcd575043aa29d39d15d879be89f7
+#Fri Jan 26 17:08:07 PST 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.1-all.zip

```

Comment:
Hmmm, the README mentions Gradle 4.5 but wrapper downloads 4.1

Reply:
I am not sure how various components are versioned. I upgraded the Gradle "plugin" in the current version of Android Studio and it gave me this version. When I downloaded the current version of Graddle itself, to verify that I did not break the command-line build, it was 4.5, and I had do use a non-latest version of the JDK to make it work, so I added a note on that to readme in case someone else hits the same problem. I see no contradiction here.

Reply:
Version mappings are listed here:
https://developer.android.com/studio/releases/gradle-plugin.html#updating-gradle

Reply:
OK, I can say that with this change, everything works perfectly correctly. Gradle 4.1 serves this change's purpose. The comment in Readme was about the "current version of Gradle" which is never than 4.1.